modules/simple_forms_api/app/models/simple_forms_api/vba_26_4555.rb

```diff
@@ -78,6 +78,12 @@ def living_situation_payload
         },
         isInCareFacility: data.dig('living_situation', 'is_in_care_facility')
       }
+
+      if Rails.env.eql?('development') || Settings.vsp_environment == 'staging'
+        payload[:careFacilityAddress][:country] = care_facility_address&.fetch('country', nil)
+      end
+
+      payload
     end
 
     def previous_hi_application_payload
```

Comment:
@rhasselle-oddball Do you mind adding some tests? Are code coverage has been hovering around the unacceptable threshold.

Reply:
@stiehlrod Pushed new tests for the payload changes!

Reply:
@stiehlrod Okay, I think we should be good now 🙏